components/module-components/contact/ContactForm.tsx

```diff
@@ -34,12 +33,17 @@ const ContactForm = ({ cfTurnstileResponse }: ContactFormProps) => {
         .required('*Пораката е задолжителна'),
     }),
     onSubmit: async (values, { resetForm }) => {
+      if (!turnstileToken) {
+        toast.error('Please complete the captcha');
```

Comment:
Since this is an invisible captcha, it doesn't make sense to ask for the user to complete a captcha which they cannot see.
Change it to something like "Unknown error, please contact administrator" or whatever @edichoska suggests